pkgs/applications/editors/emacs-modes/telega/libtgvoip.nix

```diff
@@ -0,0 +1,34 @@
+{
+pkgs
+, clangStdenv
+, libopus
+, openssl
+, libpulseaudio
+, alsaLib
+, pkgconfig
+, writeText
+, llvmPackages
+, libtool
+, writeShellScriptBin
+}:
+clangStdenv.mkDerivation {
```

Comment:
Could we not put this in `emacs-modes`? It looks generally useful for non-emacs packages.

Reply:
> Could we not put this in `emacs-modes`? It looks generally useful for non-emacs packages.

Oof, really not familliar with packaging emacs stuff. You want me to put it inside melpa overrides?

Reply:
> Is `clangStdenv` required or could we use normal `stdenv`?



That is a quick solution for libtool prefix errors.

Reply:
No I'd like it to live outside of the emacs infrastructure as it's not really emacs-specific. Probably in `pkgs/development/libraries/libtgvoip`.

Reply:
> No I'd like it to live outside of the emacs infrastructure as it's not really emacs-specific. Probably in `pkgs/development/libraries/libtgvoip`.

It's very telega specific (a fork with patches for telega), so it's useless for everything else too. I thought that if one part of telega-server is in melpa overrides, probably all the other parts should be there too.

Reply:
is it okay to put it into expression itself?